participants_logbook.md

```diff
@@ -48,3 +48,4 @@ I never felt lost. It was a great learning experience. Thanks for introducing me
 
 * This was my first summer school and I am very grateful that I attended it. The organisation was great, all the tutors very friendly and always willing to help and the materials provided extremely helpful. 
   The contents tought provide a very mighty tool and I will try to integrate it to my workflow. Many thanks! Robert Kamlah
+This was the best online school that I have attended, with excellent feedbacks and very well organized. Mainly I found examples beneficial. I am so excited and satisfied with the things I learned. And the most important thing is that it was Free! Thanks a lot. Nooshineh Fallah Zabihi
```

Comment:
Could you also add a star at the beginning of the line like everyone else?

Reply:
At the moment your line is merged into the same bullet point as the entry before you:

https://github.com/escape2020/school2021/blob/494ccac9087862d7e32e8eef6fe89f24f082923b/participants_logbook.md